product_specifications/B4tchG56_5/readme.md

```diff
@@ -0,0 +1,107 @@
+# Batch Update
+
+## Problem
+
+DOI Administrators are struggling to make changes to many DOIs at the same time, specifically when they have to make a URL migration. They are currently relying on time consuming manual processes or reaching datacite staff to for help.
+
+
+Problem validation: [We took information from open survey. The large majority of users only need this feature for url URL migrations](https://docs.google.com/document/d/17spNmeQUstzM13FrWfd4Ja18YzTKfB2v_8DSd51mJSM/edit#). 
+
+## Solution 
+
+Althought this feature initially encompassed functionality for updating any metadata field, based on the `product validation` feedback we have refocused the scope to only be used on url changes. 
+
+
+WireFrames: [Figma File](https://www.figma.com/file/YYSb7xlL2TvV5MNaRZ8tXR/Batch-Update?node-id=0%3A1)
+
+Solution Validation: [We run a feedback session with potentially interested user. These, were users that had reached Datacite in the past about processing usage using the code of practice but they found the processing heav yresource intensive for their organisations. The overall feedback is that the solution solved their needs]()
+
+## User Stories
+
+### CRITICAL
+
+- [ ] As a repository admin, I would like to be able to migrate URLs of my DOIs from fabrica in BULK, so I don't have to rely on developers to perform this task.
+
```

Comment:
Are we storing the province of the change? S
Slide 8 in the wireframes - are you saying we list the old urls?

Reply:
mmm I am not sure if I follow the question. Slide 8 shows the DOI page in Fabrica *after the change* , with the new URL. 

The provenance is captured in the activities API. But I am unsure if it answers your question